ntbs-service/Pages/Notifications/Edit/Episode.cshtml

```diff
@@ -58,7 +58,7 @@
         }
         <nhs-form-group nhs-form-group-type=@ConsultantGroupState>
             <label nhs-label-type="Standard" asp-for="Episode.Consultant">Add details of consultant</label>
-            <span ref="errorField" nhs-span-type="ErrorMessage" id="given-name-error" asp-validation-for="Episode.Consultant"></span>
+            <span ref="errorField" nhs-span-type="ErrorMessage" id="consultant-error" asp-validation-for="Episode.Consultant"></span>

```

Comment:
Not introduced in your changes, but could we please add an aria-described-by on the input to this (and again for case-manager-error)?